module2/homework/generateSequence/generateSequence.hpp

```diff
@@ -2,6 +2,9 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    std::vector<int> sequence;
+    for(int i = 0; i < count; i++) {
+        sequence.push_back((i+1)*step);
```

Comment:
Please use whitespaces around operators also here
```suggestion
        sequence.push_back((i + 1) * step);
```